### PR TITLE
fix(tokens): use hover and disabled from figma

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Enhancement

#### What is expected?

To use hover and disabled from figma and not from local. The script of sync to figma tokens has been update and I removed local overloaded values.

### CODE REVIEW

File: src/theme/tokens/dark.ts [hunk 1]
```diff
@@ -8,210 +8,163 @@ export default {
...
       textStrong: '#151a2d',
-      textStrongDisabled: '#151a2d',
+      textStrongDisabled: '#23213c',
       textStrongHover: '#151a2d',
       textWeak: '#ce80ff',
       textWeakDisabled: '#644184',
-      textWeakHover: '#e8c2ff',
+      textWeakHover: '#ce80ff',
     },
     secondary: {
```
Comment (chambo-e) [inline src/theme/tokens/dark.ts:115]: Intended to loose all secondary colours ?

**Review by chambo-e — CHANGES_REQUESTED** [review]

File: src/theme/tokens/dark.ts (hunk 1 above)
Comment (chambo-e) [reply src/theme/tokens/dark.ts:115]: Alright :)

**Comment by vincentaudebert** [comment]
lint + tests broken

**Comment by QuiiBz** [comment]
Typography colors are a lot softer, is that intentional?

Screenshot 2022-06-16 at 11 06 06

**Comment by matthprost** [comment]
> Typography colors are a lot softer, is that intentional?
> 
> Screenshot 2022-06-16 at 11 06 06

It seems like

**Review by QuiiBz — APPROVED** [review]

**Pull request merged**